services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/Hakemus.kt

```diff
@@ -33,7 +33,6 @@ sealed interface HakemusData {
     val applicationType: ApplicationType
     val name: String
     val pendingOnClient: Boolean
-    val postalAddress: PostalAddress?
```

Comment:
Why?

Reply:
Because postalAddress is not needed in all application types in Haitaton. Especially excavation notification does not have it. Excavation notification has addresses in each area but not one general address as is in cable report.

Reply:
Ok. If the change had been done when adding the excavation notification, I might've understood it right away.